BankSDK/GiniBankSDKExample/GiniBankSDKExample/ComponentAPI/ComponentAPICoordinator.swift

```diff
@@ -226,10 +226,13 @@ extension ComponentAPICoordinator {
     fileprivate func showNoResultsScreen() {
         let vc: UIViewController
         if pages.type == .image {
-            let imageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController()
-            imageAnalysisNoResultsViewController.didTapBottomButton = { [unowned self] in
+            let imageAnalysisNoResultsViewController = HelpTipsViewController(giniConfiguration: giniBankConfiguration.captureConfiguration())
```

Comment:
@chriskryn  I'm wondering why we need this change. Maybe it is better to revert it and comment out broken lines until the new NoResultsViewController will be implemented. What do you think?